src/main/views/courts/courts.njk

```diff
@@ -1,31 +1,104 @@
-{% extends "template.njk" %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
 
+{% extends "template.njk" %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Courts and Tribunals</h1>
-
-  <table class="govuk-table" id="courts">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Name</th>
-        <th scope="col" class="govuk-table__header">Last Updated</th>
-        <th scope="col" class="govuk-table__header"></th>
-        <th scope="col" class="govuk-table__header"></th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-    {% for court in courts | sort(attribute="name") %}
-      <tr class="govuk-table__row">
-        <td scope="row" class="govuk-table__cell">{{ court.name }}</td>
-        <td scope="row" class="govuk-table__cell">{{ court['updated_at'] }}</td>
-        <td scope="row" class="govuk-table__cell">
-          <a id="view-{{ court.slug }}" class="govuk-link" href="{{ factFrontendURL }}/courts/{{ court.slug }}">view</a>
-        </td>
-        <td scope="row" class="govuk-table__cell">
-          <a id="edit-{{ court.slug }}" class="govuk-link" href="/courts/{{ court.slug }}/edit#general">edit</a>
-        </td>
-      </tr>
+  <h1 class="govuk-heading-xl">Courts and tribunals</h1>
+
+  <div id = 'tableContainer'>
+    <div id='courtOptionsRow' class='govuk-grid-row'>
+      <div class='govuk-grid-column-one-half'>
+        {{ govukInput({
+          attributes: {
+            placeholder: ' Search courts and tribunals'
+          },
+          prefix: {
+            text: '🔎'
+          },
+          id: 'searchCourts',
+          name: 'searchCourts'
+        }) }}
+      </div>
+
+      <div id='toggleClosedCourtsDisplay' class='govuk-grid-column-one-quarter'>
+       {{ govukCheckboxes({
+         idPrefix: 'toggle-closed-courts-display',
+         name: 'toggleClosedCourtsDisplay',
+         classes: 'govuk-checkboxes--small',
+         items: [
+           {
+             value: 'includeClosed',
+             text: 'Include closed'
+           }]
+       }) }}
+      </div>
+    </div>
+
+    <div id='numberOfCourts' class='govuk-hint'></div>
+
+    {% set courtItems = [] %}
+    {% for court in courts|sort(false, true, 'name') %}
+      {% set courtItems = (courtItems.push([
+        {
+          attributes: {
+            type: 'name',
```

Comment:
Should we use data attributes for all of these e.g. data-name, data-displayed etc?

Reply:
Also, we should perhaps rename the slug one. Calling it 'name' might be confusing.